playground/nextjs/.eslintignore

```diff
@@ -0,0 +1 @@
+/
```

Comment:
```suggestion

```
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Maybe a typo‏

Reply:
No this is just to ignore everything, as the root one didn't work.